src/service/print/printDocx-1.0/dataImporter.js

```diff
@@ -0,0 +1,114 @@
+const { async } = require("rxjs/internal/scheduler/async");
+const fetch = require("node-fetch");
+const axios = require("axios");
+
+const envVariables = require("../../../envVariables");
+const { result } = require("lodash");
+
+class PrintDocx1ImportError {
+  constructor(message) {
+    this.message = message;
+    this.name = "PrintDocx1ImportError";
+  }
+}
+const fields =
+  "body,primaryCategory,mimeType,qType,answer,templateId,responseDeclaration,interactionTypes,interactions,name,solutions,editorState,media,name,board,medium,gradeLevel,subject,topic,learningOutcome,maxScore,bloomsLevel,author,copyright,license";
+
+const getQuestionForSet = async (id) => {
+  let url;
+  if(envVariables.ASSESSMENT_SERVICE_URL === "http://assessment-service:9000"){
+    url= `${envVariables.ASSESSMENT_SERVICE_URL}/question/v4/read/${id}?fields=${fields}`
+  }else{
+    url= `${envVariables.ASSESSMENT_SERVICE_URL}/question/v1/read/${id}?fields=${fields}`
+  }
+
+  const request = {
+    url: url,
+    method: "get",
+  };
+
+  return axios(request).then((r) => {
+    return r.data.result.question;
+  }).catch((e)=>{
+    return{
+      error:true,
+      errormsg:"wrong ID"
+    }
+  })
+};
+const getQuestionSet = async (id) => {
+  const headers = {
+    Authorization: "Bearer " + envVariables.SUNBIRD_PORTAL_API_AUTH_TOKEN
+  };
+  let url;
+  if(envVariables.ASSESSMENT_SERVICE_URL === "http://assessment-service:9000"){
```

Comment:
This is not correct way to write code

Reply:
This I have made changes like removed if the condition,